src/extension/manager/vsconanWorkspace.ts

```diff
@@ -90,6 +91,40 @@ export class VSConanWorkspaceManager extends ExtensionManager {
         this.context.subscriptions.push(this.statusBarConanVersion);
     }
 
+    private initStatusBarCommands() {
+        this.statusBarCommands[0].text = "$(cloud-download)";
+        this.statusBarCommands[0].tooltip = "Install dependencies with 'conan install'";
```

Comment:
Add 'VSConan' or 'vsconan' to indicate that the status bar button is originated from this extension `VSConan - Install dependencies ...` - to avoid confusion for users :)

Reply:
Good point! :+1